src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random numFigure = new Random();
+    private Random meaning = new Random();
+    private ColorSupplier getColor = new ColorSupplier();
```

Comment:
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
```

Давай придерживаться Java Style Guide https://mate-academy.github.io/style-guides/java/java.html#s5.2.5-non-constant-field-names